src/main/java/nextstep/ladder/view/ResultView.java

```diff
@@ -29,33 +31,32 @@ public void writeLadder(final Ladder ladder) {
 
   private String formatPlayer(final List<Player> players) {
     return players.stream()
-                  .map(player -> center(player.name(), 5, ' '))
-                  .collect(Collectors.joining(PLAYER_SEPARATION, PLAYER_PADDING, PLAYER_PADDING));
+                  .map(player -> center(player.name(), 5))
+                  .collect(Collectors.joining(repeat(SPACE, PLAYER_SEPARATION_SIZE)));
   }
 
   private String formatLine(final Line line) {
     return line.connects()
                .stream()
                .map(this::formatConnect)
-               .collect(Collectors.joining(BAR, LADDER_PADDING + BAR, BAR + LADDER_PADDING));
+               .collect(Collectors.joining(BAR, repeat(SPACE, LADDER_PADDING_SIZE) + BAR, BAR + repeat(SPACE, LADDER_PADDING_SIZE)));
```

Comment:
처음 시작은 `|` 앞에 4만큼 공백이 들어가고, 끝의 `|` 뒤에는 공백이 없습니다. :)
```suggestion
               .collect(Collectors.joining(BAR, repeat(SPACE, 4) + BAR, BAR)));
```